index.ts

```diff
@@ -668,6 +669,10 @@ commander
                   error.errors.map((err: any) => {
                     console.error(err.message);
                   });
+                  console.error(LOG.FILES_TO_PUSH);
+                   nonIgnoredFilePaths.map((filePath) => {
```

Comment:
Looks like there's an extra space here. Can you remove it?
Looks good otherwise, thanks!